clang/include/clang/StaticAnalyzer/Checkers/Checkers.td

```diff
@@ -1771,6 +1771,10 @@ def UncountedLambdaCapturesChecker : Checker<"UncountedLambdaCapturesChecker">,
 
 let ParentPackage = WebKitAlpha in {
 
+def NoUncheckedPtrMemberChecker : Checker<"NoUncheckedPtrMemberChecker">,
+  HelpText<"Check for no unchecked member variables.">,
+  Documentation<NotDocumented>;
```

Comment:
We can actually write a tiny bit of documentation that simply points the users to https://github.com/WebKit/WebKit/wiki/Safer-CPP-Guidelines (?)

Reply:
Sure, I added description based on the description for webkit.NoUncountedMemberChecker.